src/include/enclave/oe_compat.h

```diff
@@ -0,0 +1,22 @@
+#ifndef OE_COMPAT_H
+#define OE_COMPAT_H
+
+#ifdef SGXLKL_ENCLAVE
+
+#include <openenclave/corelibc/bits/types.h>
+#include <openenclave/corelibc/oemalloc.h>
+#include <openenclave/corelibc/oestdlib.h>
+#include <openenclave/corelibc/oestring.h>
+#include <openenclave/internal/safecrt.h>
+
+#define malloc oe_malloc
+#define realloc oe_realloc
+#define calloc oe_calloc
+#define free oe_free
+#define strlen oe_strlen
+#define strcmp oe_strcmp
+#define strtok_r oe_strtok_r
```

Comment:
A comment why this is done would be nice here.

Reply:
Layering. Essentially lets me use the same json parser implementation on the host and in the enclave, but I suspect there will be a different solution coming out of the relayering...

Reply:
(Comment added, marking resolved.)